qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -0,0 +1,171 @@
+import os
+import re
+import sys
+import shutil
+import argparse
+import subprocess
+
+
+# Get dpinst.exe
+def getdpinst(vol_utils):
+    if os.environ.get('PROCESSOR_ARCHITECTURE') == "AMD64":
+        dpinst_dir = "%s\dpinst_64.exe" % vol_utils
+    else:
+        dpinst_dir = "%s\dpinst_32.exe" % vol_utils
+    if not os.path.exists("C:\dpinst.exe"):
+        shutil.copy(dpinst_dir, "C:\dpinst.exe")
+    else:
+        print "dpinst.exe is already existed"
+    if not os.path.exists("C:\dpinst.xml"):
+        dpinst_xml = "%s\dpinst.xml" % vol_utils
+        shutil.copy(dpinst_xml, "C:\dpinst.xml")
+    else:
+        print "dpinst.xml is already existed"
+
+
+# Install certificate
+def certutil(vol_utils):

```

Comment:
All path in your code should be a raw string, so I think you need "r" to ask a raw string path.

Thanks,
xu